lite-pm/pages/components/Timeline.js

```diff
@@ -26,6 +28,55 @@ export default class Timeline extends Component {
     };
   }
 
+  // This likely needs to be moved @Caleb
+  addTask = () => {
+    this.toggleAddTaskModal();
+    let tempTasks = this.state.tasks;
+    tempTasks[tempTasks.length - 1] = (
+      <Task
+        key={uuid()}
+        name={this.state.taskName}
+        description={this.state.taskDescription}
+        duration={this.state.taskDuration}
+        durationType={this.state.taskDurationType} //0 for hours 1 for days
+      ></Task>
+    );
+    tempTasks.push(
+      <Button
+        key={uuid()}
+        color="secondary"
+        className={styles.addTask}
+        onClick={this.toggleAddTaskModal}
+      >
+        <FontAwesomeIcon icon={faPlus} className="mr-2" />
+        Add Task
+      </Button>
+    );
+
+    //Put the task connection code here!
+    if (this.state.taskDurationType === 0) {
+      this.state.taskDuration = this.state.taskDuration * 3600;
+    } else {
+      this.state.taskDuration = this.state.taskDuration * 86400;
+    }
+
+    const projectId = "944f27b6-e6a0-4f2b-af4b-2d3911fc7d76"; //Used for testing need to remove after for production
+    axios.post(`${baseUrl}/api/tasks/create`, {
+      projectId: projectId,
+      title: this.state.taskName,
+      duration: this.state.taskDuration,
+      description: this.state.taskDescription,
+    });
+
+    this.setState({
+      tasks: tempTasks,
+      taskName: "Name",
+      taskDescription: "Description",
+      taskDuration: 0,
+      taskDurationType: 0,
+    });
+  };
```

Comment:
This can mostly be deleted...line 56-69 can be moved into AppPage.js addTask function

Reply:
The entire `addTask` function will be deleted eventually so we can just leave it as reference for now in case you need it in `AppPage` when you are connecting it to the api